### PR TITLE
Ensure attachment contentId is a string

### PR DESCRIPTION
**Pull request by suysdavid**
While working on a project, I executed the parser on all emails in https://github.com/mikel/mail/tree/master/spec/fixtures/emails,
It failed on a mail containing no ContentID for an attachment.

### CODE REVIEW

File: tests/mails/m0028 [hunk 1]
```diff
@@ -15,83 +15,10 @@ Content-Transfer-Encoding: quoted-printable
...
-This is an attachment
---------=_MBB810AA79-F5A7-4F80-A000-3405DAF098DB
-Content-Type: text/plain; name=4BA92D8A-75A6-41A4-9A90-82AC9324077E.txt
-Content-Transfer-Encoding: base64
-Content-Disposition: attachment;
- filename=4BA92D8A-75A6-41A4-9A90-82AC9324077E.txt
-X-Part-Id: 0f46fe1047a947a993133eb3cbd89b7d
-
-This is an attachment
---------=_MBB810AA79-F5A7-4F80-A000-3405DAF098DB
```
Comment (sanmai) [inline tests/mails/m0028:89]: This seems like not related to the test added here.

**Comment by eXorus** [comment]
@suysdavid I don't understand the fix.
Did you run the parser on all the emails with the master version or the 6.0.0 version?

I don't understand the fix here on master (and master in the next major version).
Why do you change the m0028 file?
getContentId already return null, so why do you want to return empty string instead?
What si the exact error you have?

**Comment by suysdavid** [comment]
Indeed, my apologies as I was too fast to pull the trigger.
In a project I'm working on that uses this php-mime-mail-parser, I added some more example test mails (see original comment) to harden the system.
One of the files contains an attachment without an X-Part-Id.
This results in an error when fetching the content-id of the attachment, as the function Attachment::getContentID() should always return a string value, and should not be nullable.
Instead of changing the public interface for this function, I tried to fix the problem by filling it with an empty string instead

**Comment by eXorus** [comment]
Thanks but I'm not sure I would like to have an empty string.
- without content id => null
- with content id => string

So the interface should changed.

**Comment by suysdavid** [comment]
Ok, I modified the interface to make it nullable. Indeed, makes more sense!

**Comment by eXorus** [comment]
Thanks
